modules/geo-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -0,0 +1,146 @@
+import {GLTFMaterialParser} from '@luma.gl/experimental';
+import {Model, Geometry, pbr} from '@luma.gl/core';
+import {log} from '@deck.gl/core';
+import {SimpleMeshLayer} from '@deck.gl/mesh-layers';
+
+import vs from './mesh-layer-vertex.glsl';
+import fs from './mesh-layer-fragment.glsl';
+
+function validateGeometryAttributes(attributes) {
+  const hasColorAttribute = attributes.COLOR_0 || attributes.colors;
+  if (!hasColorAttribute) {
+    attributes.colors = {constant: true, value: new Float32Array([1, 1, 1])};
+  }
+  log.assert(
```

Comment:
Remove this assert. Since the user does not directly supply data to this layer, they will not be able to fix this error on their side.

Reply:
Done